libs/ui-toolkit/src/components/popover/popover.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+
+import { Intent } from '../../utils/intent';
+import { Button } from '../button';
+import { Popover } from './popover';
+
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+export default {
+  title: 'Popover',
+  component: Popover,
+} as ComponentMeta<typeof Popover>;
+
+const Template: ComponentStory<typeof Popover> = (args) => {
+  const [open, setOpen] = useState(args.open);
+  return (
+    <div>
+      <Popover
+        {...args}
```

Comment:
Question (non-blocking): We're beginning to become more strict about what props ui-toolkit components can receive. For example, we probably don't want consumers to be able to add any old Tailwind class. Is it possible to control props more carefully?

Reply:
updated. I think this is what you mean?

Reply:
@MadalinaRaicu not exactly, sorry I wasn't clearer. I don't think we should be spreading args on line 18. Instead, we should control what args are possible with this component. For example, if there are a narrow set of styling types allowed by design, we should have specific props for these and control styling ourselves, rather than allowing someone to use the className component however they please to affect how it looks.

Reply:
I did remove ` {...args}` entirely, the file has been updated, this comment is still on outdated version